### PR TITLE
Add missing property details for AesManaged

### PR DESCRIPTION
**Pull request by MSDN-WhiteKnight**
## Summary

Add missing property details for AesManaged.

### CODE REVIEW

**Comment by rpetrusha** [comment]
@bartonjs, could you review this PR, please?

**Comment by mairaw** [comment]
Internal review link: https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.aesmanaged?branch=pr-en-us-2214&view=netframework-4.7.2

File: xml/System.Security.Cryptography/AesManaged.xml [hunk 1]
```diff
@@ -690,8 +706,12 @@
       <Docs>
         <summary>Gets or sets the padding mode used in the symmetric algorithm.</summary>
         <value>One of the enumeration values that specifies the type of padding to apply. The default is <see cref="F:System.Security.Cryptography.PaddingMode.PKCS7" />.</value>
-        <remarks>To be added.</remarks>
+        <remarks><format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Padding is used when there's not enough data to fill the last block. See the <xref:System.Security.Cryptography.PaddingMode> enumeration for a description of specific modes.
```
Comment (bartonjs) [inline xml/System.Security.Cryptography/AesManaged.xml]: Seems like this should re-use the value from the SymmetricAlgorithm base class, or just seealso to SymmetricAlgorithm's version of it.

File: xml/System.Security.Cryptography/AesManaged.xml [hunk 2]
```diff
@@ -490,7 +494,11 @@
       <Docs>
         <summary>Gets or sets the initialization vector (IV) to use for the symmetric algorithm.</summary>
         <value>The initialization vector to use for the symmetric algorithm</value>
-        <remarks>To be added.</remarks>
+        <remarks><format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The symmetric encryption algorithms operating in Cipher Block Chaining (CBC) mode combine the block with the portion of data from the previous block in the process of encryption. When encrypting the first block, initialization vector is used instead of the previous block value. The initialization vector size must be the same as the block size.
```
Comment (bartonjs) [inline xml/System.Security.Cryptography/AesManaged.xml]: Seems like this should copy, or point to, SymmetricAlgorithm's definition of this.

File: xml/System.Security.Cryptography/AesManaged.xml [hunk 3]
```diff
@@ -105,9 +105,13 @@
...
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the block size, in bits, of the cryptographic operation.</summary>
+        <value>The block size, in bits, of the cryptographic operation. The default is 128 bits.</value>
+        <remarks><format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The block size is the basic unit of data that can be encrypted or decrypted in one operation. Messages longer than the block size are handled as successive blocks; messages shorter than the block size must be padded with extra bits to reach the size of a block. For AES, the only valid block size is 128 bits.
```
Comment (bartonjs) [inline xml/System.Security.Cryptography/AesManaged.xml]: Seems like this should copy, or point to, SymmetricAlgorithm's definition of this.

**Comment by MSDN-WhiteKnight** [comment]
@bartonjs  @rpetrusha I've made changes according to the review

**Pull request merged**